.github/workflows/ci.yml

```diff
@@ -2,16 +2,18 @@ name: ci
 on:
   push:
     branches: ['*']
-  pull_request:
```

Comment:
Deleting this means that we no longer build pull requests (coming from a fork) raised against the repository.